autospec/specfiles.py

```diff
@@ -1162,3 +1163,35 @@ def _write(self, string):
 
     def _write_strip(self, string):
         self.specfile.write_strip(string)
+
+    @staticmethod
+    def quote_filename(filename):
+        """
+        Quotes the filename, if necessary. Identifies and skips any RPM directive prefix.
+        """
+        # Characters that require quoting -- only those with special 
+        # meaning in specfiles
+        special_chars = set(" \t")
+        # Build up the output as a string
+        quoted = ''
+        # Separate any directive prefix
+        # This will break if your directive contains a slash!
+        line = filename.split("/", 1)
+        if len(line) < 1:
+            print_fatal("Filenames must start with a / ({0})".format(filename))
+            sys.exit(1)
+        elif len(line) > 1:
+            # Keep the prefix
+            quoted += line[0]
+            # Filename is remaining part
+            filename = line[1]
+        else:
+            filename = line[0]
+        # Now check for special characters
+        if any(c in filename for c in special_chars):
+            # Quote the remaining filename, with leading slash
+            quoted += "\"/{}\"".format(filename)
```

Comment:
Instead of escaping the quotes with `"\"...\""` I would prefer to use single quotes, like this:
```
quoted += '"/{}"'.format(filename)
```
since I think it is easier to read. Same goes for the tests below.

Reply:
Agreed; fixing.